commands/cluster-links.md

```diff
@@ -0,0 +1,49 @@
+Each node in a Redis Cluster maintains a pair of long-lived TCP links with each
+peer in the cluster, one established toward the peer, another accepted from the
+peer.
```

Comment:
New style is to avoid added line breaks, since markdown will remove them anyways. Also, although what you said is technically true about inbound/outbound, semantically they are for inbound/outbound.
```suggestion
Each node in a Redis Cluster maintains a pair of long-lived TCP link with each peer in the cluster: One for sending outbound messages towards the peer and one for receiving inbound messages from the peer.
```